buddyboss/buddyboss-platform
# PROD-816  Fix : The target attribute in Paragraph Text field is being removed after saving

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2414 .

### How to test the changes in this Pull Request:

1. Add a Paragraph Text field in a profile fieldset
2. Go to the frontend and edit your profile
3. Edit a text and insert a link
4. Make sure to enable "Open link in a new tab".
5. Save changes and notice that the anchor attributes including the target attribute are missing

### Proof Screenshots or Video

https://www.loom.com/share/ba65cfc0e3e0412abde1959cf6a14022

### Other information:
